src/me/mrCookieSlime/Slimefun/listeners/BackpackListener.java

```diff
@@ -41,27 +45,41 @@ public void onClose(InventoryCloseEvent e) {
 	}
 	
 	@EventHandler
+	public void onItemDrop(PlayerDropItemEvent e) {
+		if (Variables.backpack.containsKey(e.getPlayer().getUniqueId())){
+			ItemStack item = e.getItemDrop().getItemStack();
+			SlimefunItem sfItem = SlimefunItem.getByItem(item);
+			if (sfItem instanceof SlimefunBackpack || sfItem instanceof SoulboundBackpack) e.setCancelled(true);
```

Comment:
Okay yes, it does.
https://github.com/TheBusyBiscuit/Slimefun4/blob/6f61176d0c9cb944976e95813160a8df0759effc/src/me/mrCookieSlime/Slimefun/listeners/DamageListener.java#L55